tools/reach-calculator.html

```diff
@@ -0,0 +1,233 @@
+            box-shadow: inset 0 -4px 4px -4px #4d4d4d;
+            color: #4d4d4d;
+            cursor: pointer;
+            height: 2em;
+            padding: 0 1em;
+        }
+        .domain #domainButton, .api #apiButton { /* When tab button is active */
+            background: #0000c0;
+            border-color: #0000c0;
+            box-shadow: none;
+            color: #fff;
+        }
+        .domain #apiButton, .api #domainButton {
+        }
+        .domain #apiTab, .api #domainTab {
+            display: none;
+        }
+        .domain #domainTab, .api #apiTab {
+            display: block;
+        }
+        #calculator {
+            border: solid 4px #0000c0;
+            border-radius: 0 8px 8px 8px;
+            padding: 1em;
+        }
+        #calculator div {
+            margin-bottom: 1em;
+        }
+        input[type="number"] {
+            width: 8em;
+        }
+        hr {
+            margin: 1em 0;
+        }
+        #results {
+            font-weight: bold;
+        }
+    </style>
+</head>
+<body class="domain">
+<div id="container">
+    <h2>Site Breakage Reach Calculator</h2>
+
+    <div id="options">
+        <button id="domainButton" class="tablink">Domains</button>
+        <button id="apiButton" class="tablink">APIs/Features</button>
+    </div>
+
+    <div id="calculator">
+        <div id="domainTab" class="optionTab">
+            <label>Domain rank from <a href="https://tranco-list.eu/query" target="_blank">Tranco</a>:<br>
+            <input type="number" id="domainRankInput" min="1" max="100000000"></label>
+            <div id="domainPopularityOutput"></div>
+        </div>
+
+        <div id="apiTab" class="optionTab">
+            <label>API/feature usage share (%):<br>
+            <input type="number" id="apiUsageInput" min="0" max="100"></label>
+            <div>Note: <a href="https://chromestatus.com/metrics/feature/popularity" target="_blank">Chrome Platform Status</a> is useful for JS APIs.</div>
+        </div>
+
+        <div>
+            <label><abbr title="Further reduction assumption, e.g. visiting 2+ pages, particular version, etc.">[Optional] Reduction assumption (%):<br>
+            <input type="number" id="reductionFactorInput" min="0" max="100"></abbr></label><br>
+        </div>
+
+        <div>
+            <label for="confidenceInput">Additional unaccounted-for breakage?</label><br>
+            <label><input type="radio" id="low" name="confidenceInput" value="low" checked> Low confidence</label><br>
+            <label><input type="radio" id="medium" name="confidenceInput" value="medium"> Medium confidence</label><br>
+            <label><input type="radio" id="high" name="confidenceInput" value="high"> High confidence</label><br>
+        </div>
+    
+        <hr>
+        <div id="results"></div>
+        <div>(Use this and the Severity Assessment Matrix to get a priority level.)</div>
+    </div>
+</div>
+
+<script>
+(function () {
```

Comment:
I believe this repo never turned on HTML embedded linting of JS, so this all isn't being checked for formatting against eslint.

Can we just throw this into another file? (then run `npx eslint --fix` etc)

Reply:
Skipping this due to linter seeming to work with embedded JS.